subprojects/distributions/src/changes/accepted-public-api-changes.json

```diff
@@ -145,6 +145,14 @@
             "member": "Method org.gradle.kotlin.dsl.ComponentMetadataHandlerExtensionsKt.withModule(org.gradle.api.artifacts.dsl.ComponentMetadataHandler,java.lang.Object,org.gradle.api.Action)",
             "acceptation": "Reified versions of methods that are already de-incubated",
             "changes": []
+        },
+        {
+            "type": "org.gradle.api.tasks.JavaExec",
```

Comment:
I think once a method is annotated with incubating, you don't need to accept a public API change.